### ISSUE TITLE
experience report: issuer.js

### ISSUE THREAD

**Issue by dckc** [agoric-labs/jessica#34]
comparing [issuers.js][1] vs. the grammar here (via [a scala implementation][2]), I ran across

 - #32 `import './types';`
 - `new Error(...)` which, I gather, can be rewritten as `Error(...)` with no observable change
 - `new Set()` (as discussed earlier in https://github.com/Agoric/Jessie/issues/29 etc.)
 - #33 `export { makeIssuerKit };` as well as module level function declarations

To conform to the current grammar we'd have to change...

```js
function makeIssuerKit() { ... }
harden(makeIssuerKit);
export { makeIssuerKit };
```

to more like...

```
export const makeIssuerKit = harden((...) => {...});
```

[1]: https://github.com/Agoric/agoric-sdk/blob/7db7e5c4c569dfedff8d748dd58893218b0a2458/packages/ERTP/src/issuer.js
[2]: https://github.com/rchain-community/js2rho/blob/a606757e61bc10ebf3d576b77ec1a9695b8e5c51/src/main/scala/js2rho/JessieParser.scala

cc @erights @michaelfig

**Comment by erights** [comment]
For all four of these, I think we should change the definition of Jessie to allow them. I'll comment on each of the issues you link to.

**Comment by erights** [comment]
No, after reading https://github.com/Agoric/Jessie/issues/29 I changed my mind. It is correct. We should implement and use that std library so that we don't have unnecessarily unhardened objects, like instances of `Set` running around loose.
